### PR TITLE
help panel implementation

### LINKED ISSUE #11: Create a help panel
Add a panel that contains the list of keybindings with an explanation on what they do.

The panel would occupy on the whole screen, and its visibility would be toggled using a key (such as `h`).

### PR DESCRIPTION
**Pull request by Siddhi-agg**
Resolves #11. As advised by you, I have created a new PR because of the merge conflicts in the older one #35 .

### CODE REVIEW

File: lib/interactive_viewer.ml [hunk 1]
```diff
@@ -104,6 +105,16 @@ let change_summary z_patches : ui Lwd.t =
   W.string ~attr:Notty.A.(fg lightcyan) operation_count
 
 let view (patches : Patch.t list) =
+  let help_panel =
+    Ui.vcat
+      [
+        W.string "Help Panel:\n";
+        W.string "h:   Toggle the help panel";
```
Comment (panglesd) [inline lib/interactive_viewer.ml:112]: `h` does not actually _toggle_ the help panel. Instead, one has to use `q` to get out of the help panel.

I would prefer the "toggling" mechanism advertised than the "`q` to quit help panel" currently implemented!

**Review by panglesd — APPROVED** [review]
Other than the small comments below, this is very good! I'll merge as soon as you've addressed the comments.

File: lib/interactive_viewer.ml (hunk 1 above)
Comment (Siddhi-agg) [reply lib/interactive_viewer.ml:112]: @panglesd In my last PR for the same issue, I was asked to set ```q``` for exiting the help panel by @Julow. So do you want me to set ```h``` for toggling the help panel and when the help panel is open, ```q``` to do nothing ?

Comment (panglesd) [reply lib/interactive_viewer.ml:112]: Oh, yes he did say it! Sorry for the disorder.
Let's keep what @Julow told you to do! However, let's have the description match. So `h` should be described as "opening" not "toggling" the help panel.

**Comment by Siddhi-agg** [comment]
@panglesd I have implemented the changes you requested. Hope the PR is good now :)

**Review by panglesd — APPROVED** [review]
Thanks a lot for your contribution!

**Pull request merged**